### DISCUSSION TITLE
猫又组队

### DISCUSSION THREAD

**Discussion by Alsuhold** [share121/inter-knot#3320] [其他]
猫又，安比，第三个能用可琳么，具体咋搞<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >
![Uploading 1703474550906a4f2e16ea1fb99455ccd1feb471d5d5450ae1d49c1f7e684c35545463105a0cf.0.jpg…]()

**Comment by opqphp** [comment]
能用可琳（图片加载不出来
